ios/Classes/ComAppceleratorUrlSessionSessionProxy.m

```diff
@@ -25,6 +25,15 @@ - (id)_initWithPageContext:(id<TiEvaluator>)context andArguments:(NSDictionary*)
     
     return self;
 }
+-(void)_initWithProperties:(NSDictionary *)properties
+{
+    sessionConfiguration = [properties objectForKey:@"configuration"];
+    [self rememberProxy:sessionConfiguration];
+    session = [NSURLSession sessionWithConfiguration:[sessionConfiguration sessionConfiguration]
+                                            delegate:(id<NSURLSessionDelegate>)[[UIApplication sharedApplication] delegate]
+                                       delegateQueue:nil];
+    [super _initWithProperties:properties];
```

Comment:
The `_initWithPageContext:` should be invoked in [here](https://github.com/rlustemberg/ti.urlsession/blob/30989c2ac0803fd0b96d7f4648c0fa0df53cd163/ios/Classes/ComAppceleratorUrlSessionModule.m#L76) from the JS side. Can you give me a JS example that won't call it?